en/docs/quick-start-guide/getting-started/deploy-siddhi-application.md

```diff
@@ -1,19 +1,19 @@
 # Step 3: Deploy the Siddhi Application
 
-The `SweetFactoryApp` that you created in [Step 2: Create the Siddhi Application](create-the-siddhi-application.md) is now ready to be deployed in the Streaming Integrator server, exported as a Docker image, or deployed in Kubernetes.
```

Comment:
Direct deploying of siddhi application to server using the vscode extension is not available. We initially had this and decided to remove it because it is a part of CI, CD not tooling

Reply:
Removed this section